docs/source/user_guide/troubleshooting.md

```diff
@@ -0,0 +1,32 @@
+# Troubleshooting guide
+
+## "DNS domain=`your_qhub_domain` record does not exist"
+
+### *Issue*
+During your initial QHub deployment, at the end of "04-kubernetes-ingress" stage, you receive an output message stating that the DNS record for `your_qhub_domain` "appears not to exist, has recently been updated, or has yet to fully propagate."
```

Comment:
```suggestion
During your initial QHub deployment, at the end of the `04-kubernetes-ingress` stage, you receive an output message stating that the DNS record for `your_qhub_domain` "appears not to exist, has recently been updated, or has yet to fully propagate."
```

in the 04-kubernetes.... link to the relevant section in the docs

Reply:
This is referencing one of the deployment stages, I don't think we have docs that discuss in detail what each of the stages do.